docs/consensus.md

```diff
@@ -92,16 +92,16 @@ use.
 This command will create and send appropriate transaction to the network and
 you should then wait for it to settle in a block. If all goes well it'll end
 with "HALT" state and your registration will be completed. You can use
-`query tx` command to see transaction status or
-`getnextblockvalidators` to see if your candidate was added.
+`query tx` command to see transaction status or `query candidates` to see if
```

Comment:
A couple of words about `query committee`?

Reply:
It doesn't make sense here, but you're right in that cli.md should be updated.

Reply:
Fixed.